packages/block-editor/src/components/url-input/index.js

```diff
@@ -84,12 +84,13 @@ class URLInput extends Component {
 			}, 100 );
 		}
 
-		// Only attempt an update on suggestions if the input value has actually changed.
 		if (
 			prevProps.value !== value &&
-			this.shouldShowInitialSuggestions()
+			! this.props.disableSuggestions &&
+			! this.isUpdatingSuggestions &&
+			! this.props.__experimentalShowInitialSuggestions
```

Comment:
```suggestion
			! this.props.disableSuggestions &&
			! this.isUpdatingSuggestions
```

In my testing I don't think we need `! this.props.__experimentalShowInitialSuggestions`. 

Why?

Because this property is designed to determine whether not to show suggestions immediately (before any typing) upon mounting the control. If you turn the `__experimentalShowInitialSuggestions` to `true` then you will see that when you create a new link suggestions immediately appear as soon as the Link UI opens.

You can try this by adding the property `showInitialSuggestions={ true }` to:

https://github.com/WordPress/gutenberg/blob/576f977c405d0d5d5ead0a66c6a8d9c1bef16a06/packages/format-library/src/link/inline.js#L224

All the `<LinkControl>` tests continue to pass so I'm 🤞 that this won't have knock on effects.